src/components/MdMenu/MdMenu.vue

```diff
@@ -30,6 +30,10 @@
         type: Boolean,
         default: true
       },
+      mdCloseOnClick: {
+        type: Boolean,
+        default: true
```

Comment:
I think this should be false on start. That's because we may have a menu that does not implement lists inside. For some "filter" panels for example.